ts/util/attachmentsUtil.ts

```diff
@@ -189,44 +189,52 @@ export async function autoScale<T extends { contentType: string; blob: Blob }>(
     throw new Error(`GIF is too large, required size is ${maxSize}`);
   }
 
-  const loadImgOpts: LoadImageOptions = {
-    maxWidth: makeSquare ? maxMeasurements?.maxSide : maxWidth,
-    maxHeight: makeSquare ? maxMeasurements?.maxSide : maxHeight,
-    crop: !!makeSquare,
-    orientation: 1,
-    aspectRatio: makeSquare ? 1 : undefined,
-    canvas: true,
-    imageSmoothingQuality: 'medium',
-  };
-
   perfStart(`loadimage-*${blob.size}`);
-  const canvas = await loadImage(blob, loadImgOpts);
+  const canvasPlop = await loadImage(blob, {});
+  const canvasOri = loadImage.scale(
+    canvasPlop.image, // img or canvas element
+    {
+      maxWidth: makeSquare ? maxMeasurements?.maxSide : maxWidth,
+      maxHeight: makeSquare ? maxMeasurements?.maxSide : maxHeight,
+      crop: !!makeSquare,
+      cover: !!makeSquare,
+      orientation: 1,
+      // aspectRatio: makeSquare ? 1 : undefined,
+      canvas: true,
+      imageSmoothingQuality: 'medium',
+      meta: false,
+    }
+  );
+  if (maxMeasurements?.maxSide) {
```

Comment:
This needs cleanup or a comment to explain when we need this for debugging purposes

Reply:
removed